src/components/MasterDetails.js

```diff
@@ -124,7 +129,7 @@ type MasterDetailsDefaultProps = {
 type MasterDetailsState = {
   /** The IDs of the selected rows */
```

Comment:
It might be useful to add a comment here about selectedRows having the rows being in chronological order with the most recently selected one at the end of the list... (if this is still a true statement, of course).

Reply:
With multiselect, determining the active row is now more complex than chronological order.
A row can only become the active row if it was clicked without a modifier.
A row can only become an anchor row if it was clicked without using the shift key.
Because of this, the details row is tracked separately. Pulling out the oldest selected row won't tell you how that key was clicked. It's currently selected, but it may never have qualified as an active row - not if `noEmptySelection` is `false`. Instead, the `detailsRowId` is updated whenever a qualifying click is made. This is an index driven multiselect. Chunks of rows may be selected at a time. Directionality is determined by the anchor row. Chronological order is no longer the driving factor and is not guaranteed as Sets are not ordered and cannot guarantee order.